internal/app/app.go

```diff
@@ -110,7 +116,7 @@ func (a *App) run(cCtx *cli.Context) error {
 		return fmt.Errorf("load datasources: %s", err)
 	}
 
-	inputFiles, err := a.compileGlob(cCtx.String("input"))
+	inputFiles, err := a.compileGlob(cCtx.String("input"), cCtx.String("exclude"))
```

Comment:
don't pass it to the compileGlob function as it's not related.

call the compileGlob function again to get the exclude results and then calculate the diff here

Reply:
:(